data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/AggregateProcessorConfig.java

```diff
@@ -32,6 +32,9 @@ public class AggregateProcessorConfig {
     @NotNull
     private Boolean localMode = false;
 
+    @JsonProperty("allow_raw_events")
```

Comment:
I think this is a rather confusing name for a couple of reasons: 1) What is a "raw" event, and 2) allow indicates that something is permitted to be used.

We are adding a feature to also "output the input event." Perhaps: `output_unaggregated_events` or `output_original_events`.